packages/core/docs/commercetools/setup-store.md

```diff
@@ -0,0 +1,309 @@
+
+Then, enter specific methods for each zone and currency:
+
+<center>
+  <img src="./images/setup-store/shipping-methods_1.png" alt="Commercetools shipping methods configuration" class="image-shadow preview-view" />
+</center>
+
+## Adding a new category
+
+Categories allow to organize the products in groups and make them easy to find by the customer. See the [Modeling Categories](https://docs.commercetools.com/tutorials/product-modeling/categories) page to know more about categorizing items in your store.
+In order to create a category you need to:
+
+1. Choose `add category` from the sidebar menu. 
+  <center>
+    <img src="./images/setup-store/categories-sidebar-signed.png" alt="Commercetools create category first step" class="image-shadow" />
+  </center>
+
+2. Enter the name, and category order.
+  <center>
+    <img src="./images/setup-store/create-category-0.png" alt="Commercetools create category first step" class="image-shadow preview-view" />
+  </center>
+3. If applicable, select the parent category to which this category belongs.
+  <center>
+    <img src="./images/setup-store/create-category-1.png" alt="Commercetools create category second step" class="image-shadow preview-view" />
+  </center>
+
+4. Enter the slug. It will be part of the URL for this category.
+  <center>
+    <img src="./images/setup-store/create-category-2.png" alt="Commercetools create category third step" class="image-shadow preview-view" />
+  </center>
+
+Please provide category structure with at least two levels, in order to show category page properly in VSF 2 project (one is on top, inside main menu and second level categories are shown in menu on the left).
+
+You can see your categories in `categories search` menu:
+  <center>
+    <img src="./images/setup-store/categories-search.png" alt="Commercetools create product type first step" class="image-shadow preview-view" />
+  </center>
+
+## Adding a new product type
+
+To create any product in commercetools, at least one product type needs to be set. It is a template with specific attributes for products of the same type. You can read more about them on the [Modeling Product Types and Attributes](https://docs.commercetools.com/tutorials/product-modeling/product-types) page. Creating a product type involves changes in `Product types` menu:
+
+
+  <center>
+    <img src="./images/setup-store/product-types-sidebar-signed.png" alt="Commercetools create product type first step" class="image-shadow" />
+  </center>
+
+1. Enter name and description.
+
+  <center>
+    <img src="./images/setup-store/product-type-1.png" alt="Commercetools create product type first step" class="image-shadow preview-view" />
+  </center>
+
+
+2. Click the 'Add an attribute' button and enter the name, label, constraints, and type fields.
```

Comment:
```suggestion
2. Click the `Add an attribute` button and enter the name, label, constraints, and type fields.
```
Maybe use code-styling for thinks like that, just like you did in some other places?

Reply:
Of course